rest/route/host_agent_test.go

```diff
@@ -857,295 +858,674 @@ func TestTaskLifecycleEndpoints(t *testing.T) {
 	}
 }
 
-func TestAssignNextAvailableTaskWithDispatcherSettingsVersionTunable(t *testing.T) {
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
+func TestAssignNextAvailableTaskNew(t *testing.T) {
```

Comment:
does this need to have new in the title?

Reply:
The old one was because we had two dispatcher settings but now we only have one, I can revert it but I thought that the old one is just an overload of information and doesn't help debug then the shorter one

Reply:
Sorry I don't understand -- can we just call this "TestAssignNextAvailableTask"?  It doesn't seem like we have anything else called that; I agree with pairing down the name but I don't see any reason to specify "new"

Reply:
I misread your question, we don't need new (it was just from me testing different implementations!